backend/analytics_server/mhq/api/pull_requests.py

```
@@ -167,3 +167,38 @@ def get_team_lead_time_trends(
+@queryschema(
+    Schema({
+        Required("from_time"): All(str, Coerce(datetime.fromisoformat)),
+        Required("to_time"):   All(str, Coerce(datetime.fromisoformat)),
+        Optional("pr_filter"): All(str, Coerce(json.loads)),
+    })
+)
+def get_prs_merged_without_review(
+    team_id: str,
+    from_time: datetime,
+    to_time: datetime,
+    pr_filter: Dict = None,
```

Ruff (0.14.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)